.github/scripts/trymerge.py

```diff
@@ -1113,6 +1113,13 @@ def gen_commit_message(
             msg_body = re.sub(RE_GHSTACK_DESC, "", msg_body)
         msg = self.get_title() + f" (#{self.pr_num})\n\n"
         msg += msg_body
+
+        # Mention PR co-authors
+        for author_login, author_name in self.get_authors().items():
+            if author_login == self.get_pr_creator_login():
+                continue
+            msg += f"\nCo-authored-by: {author_name}"
```

Comment:
Maybe this is simpler?

```suggestion
            if author_login != self.get_pr_creator_login():
                msg += f"\nCo-authored-by: {author_name}"
```